iac/terraform/aws/web-client/cloudfront.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  #checkov:skip=CKV_AWS_68: "CloudFront Distribution should have WAF enabled"
```

Comment:
as we are not fixing this, please do raise a ticket for this and we will pick this up later

Reply:
https://github.com/CraftsmenLtd/BloodConnect/issues/120
Done creating